.travis.yml

```diff
@@ -7,3 +10,11 @@ script:
 - npm run generate-coverage
 - npm run check-coverage
 - npm run upload-coverage-coveralls
+
+after_script:
+- docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+- chmod +x scripts/*
```

Comment:
Yeah I'm happy to do that, Windows has bitten me a few times with it when I've been moving/renaming the scripts, and reverting the 755 permissions back to 644.